client/src/components/Header/index.js

```diff
@@ -30,14 +30,25 @@ const Header = () => {
             <NavItem>
               <RouterLink to="/registration">Registration</RouterLink>
             </NavItem>
-            <NavItem>
-              <RouterLink to="/cabinet">Cabinet</RouterLink>
-            </NavItem>
+            {(() => {
+              if (id) {
+                return (
+                  <NavItem>
+                    <RouterLink to="/cabinet">Cabinet</RouterLink>
+                  </NavItem>
+                );
+              }
+              return null;
+            })()}
```

Comment:
Face to face (f2f).

Reply:
https://github.com/dimapanasiuk/mern/commit/951983e6e67c2597c71e7a99a9bbc1b273647e58